README.md

```diff
@@ -42,6 +42,65 @@ The Inferno and CQF Ruler containers will be able to communicate, as they are on
 For STU3, the base URL is `http://<host>:8080/cqf-ruler-dstu3/`.
 For R4, the base URL is `http://<host>:8080/cqf-ruler-r4/`.
 
+### Preloading CQF Ruler
+
+To generate the cqf-ruler docker image with data already on it, the following
+steps were taken:
+
+1. Run the base cqf-ruler docker container:
+
+```
+docker run --name cqf-ruler --rm -dit -p 8080:8080 contentgroup/cqf-ruler
+```
+
+2. Wait for the server to initalize and be able to accept HTTP requests:
+
+```
+until `curl --output /dev/null --silent --head --fail http://localhost:8080/cqf-ruler-dstu3`; do printf '.'; sleep 5; done
+```
+
+3. Clone the DBCG/connectathon repository. Note that a specific commit was used
+   so that the filepaths are predicable, as they chance once in a while.
+
+```
+git clone https://github.com/DBCG/connectathon.git
+cd connectathon && git checkout 425795fb8dd39e213140493f22e77880a9257f00
+```
+
+4. Post the resources desired to be present on the cqf-ruler image:
+
+```
+# EXM165 Measure
+curl -X POST \
+http://localhost:8080/cqf-ruler-dstu3/fhir/Measure \
+-H 'Content-Type: application/json' \
+-d @connectathon/fhir3/resources/measure/measure-EXM165_FHIR3-8.5.000.json
+
+# Libraries
+curl -X POST \
+http://localhost:8080/cqf-ruler-dstu3/fhir \
+-H 'Content-Type: application/json' \
+-d @connectathon/fhir3/resources/library/all-libraries-bundle.json
+
+# Valuesets
+curl -X POST \
+http://localhost:8080/cqf-ruler-dstu3/fhir \
+-H 'Content-Type: application/json' \
+-d @connectathon/fhir3/resources/valuesets/valuesets-EXM165_FHIR3-8.5.000.json
```

Comment:
I think we should exclude the Valuesets. I don't think we will need them in our inferno tests and they'd prevent us from publicly sharing this docker image.

Reply:
Not that I disagree that we should exclude them, but our Inferno tests would definitely need these to be present if the tests do any type of calculation (i.e. $evaluate-measure, $cql)

Reply:
Oh right I guess I was just thinking about submit-data really. How about we omit ValueSets for now so that we can distribute our environment at the Jan c-thon and table the concern about evaluate-measure for now. It’s easy enough to add back in if need be.

Reply:
Should we have 2 separate images, one with valuesets and one without?  Current commit points to an image with valuesets--can either remove from that or make a separate one once we decide

Reply:
Separate works, just give it a different tag

Reply:
Added image without valuesets and updated docker-compose.yml, should be good to re-review